_templates/footer.html

```diff
@@ -1,12 +1,13 @@
 {% extends "!footer.html" %}
 
 {%- block extrafooter %}
+
 <div class='share-button-container'>
     <div class="socialbutton-wrapper">
         <a href="https://twitter.com/share?url={{ pageurl|e }}&hashtags=nuitka" target="_blank">
             <div class="socialbutton twitter" data-share-url="https://nuitka.net">
                 <p class="icon">
-                    <img src="/_static/icon-twitter.png">
+                    <img src="../_static/icon-twitter.png">
```

Comment:
This was wrong as explained, and should not be necessary with the new URL anymore.

Reply:
Sphinx can handle this without a problem.

Reply:
Using `/` can confuse Sphinx and complicate matters.

Reply:
I don't see how it could, Sphinx never sees that code again, but `/` is working fine now, whereas `..` will not with page folders.

Reply:
Or let me be more clear, this ends up directly in every generated page, independent of how it is located relative to the root folder, and therefore sometimes it would have to be `..`, other times it would be `.` and yet other times it would be `../..` and I am not going to bother with relative paths when I can use absolute ones with no problem. 

In theory, using `pageurl` you could construct a relative URL in Jinja2 there, but why bother. With the subdomain `translation` this is all solved, so please revert that part as it's useless now.

Reply:
For `translation` with `/`, the icon does not display properly.

Reply:
Please revert it, then I can see it and potentially fix it. All I can see now is that the social buttons do not work in the URL I have given. And I told you, I know there is no way for it to be correct on all pages this way.